### PR TITLE
[NEW] Added Slovene (sl-SI) translation.

### PR DESCRIPTION
**Pull request by cbuctok**
@RocketChat/core

### CODE REVIEW

File: packages/rocketchat-i18n/i18n/sl-SI.i18n.json [hunk 1]
```diff
@@ -1,3 +1,1721 @@
 {
-  "#channel": "#kanal"
-}
\ No newline at end of file
+    "en" : {
```
Comment (ggazzo) [inline packages/rocketchat-i18n/i18n/sl-SI.i18n.json]: delete this line please

File: packages/rocketchat-i18n/i18n/sl-SI.i18n.json [hunk 2]
```diff
@@ -1,3 +1,1721 @@
...
+	"Logout": "Logout",
+	"Logout_Others": "Odjava iz drugih prijavljenih lokacij",
+	"Mail_Message_Invalid_emails": "Predložili ste eno ali več neveljavnih e-pošnih sporočil:% s",
+	"Mail_Message_Missing_to": "Izbrati morate enega ali več uporabnikov ali zagotoviti enega ali več e-pošnih naslovov, ločenih z vejicami.",
+	"Mail_Message_No_messages_selected_select_all": "Niste izbrali nobenega sporočila. Želite <a href='#'class='selectallall\\> izberite vse </ a> vidna sporočila?",
+	"Mail_Messages": "Sporočila poše",
+	"Mail_Messages_Instructions": "Izberite sporočila, ki jih želite poslati po e-poši, tako da kliknete sporočila",
+	"Mail_Messages_Subject": "Tukaj je izbran del% s sporočil",
+	"Mailer": "Mailer",
+	"Mailer_body_tags": "<B> mora </ b> upor
```
Comment (ggazzo) [inline packages/rocketchat-i18n/i18n/sl-SI.i18n.json]: this line too :)

Comment (ggazzo) [inline packages/rocketchat-i18n/i18n/sl-SI.i18n.json]: delete this line too please :P

**Comment by ggazzo** [comment]
@cbuctok hey, I'm really interested with your contribution, do you need some help?

**Comment by cbuctok** [comment]
Hey @ggazzo I removed the repo around a half a year ago So I do not have this file anymore.
I manually moved everything to linguohub and it was enough time invested for me for this project. I wish you all the best!

**Pull request closed without merging**